api/pom.xml

```diff
@@ -214,9 +229,6 @@
                 <configuration>
                     <archive>
                         <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
```

Comment:
Probably out of scope for this PR, but we really should also add the OSGi headers in the manifest.

Reply:
The `/api/` build here already adds OSGi manifest headers.

The result of `mvn clean install` on `master` branch today.

```
Manifest-Version: 1.0
Archiver-Version: Plexus Archiver
Created-By: Apache Maven Bundle Plugin
Built-By: joakim
Build-Jdk: 11.0.6
Automatic-Module-Name: java.servlet
Bnd-LastModified: 1586271343407
Bundle-Description: Jakarta Servlet 5.0
Bundle-DocURL: https://www.eclipse.org
Bundle-License: http://www.eclipse.org/legal/epl-2.0, https://www.gnu.or
 g/software/classpath/license.html
Bundle-ManifestVersion: 2
Bundle-Name: Jakarta Servlet
Bundle-SymbolicName: jakarta.servlet-api
Bundle-Vendor: Eclipse Foundation
Bundle-Version: 5.0.0
Export-Package: jakarta.servlet;uses:="jakarta.servlet.annotation,jakart
 a.servlet.descriptor";version="5.0.0",jakarta.servlet.annotation;uses:=
 "jakarta.servlet";version="5.0.0",jakarta.servlet.descriptor;version="5
 .0.0",jakarta.servlet.http;uses:="jakarta.servlet";version="5.0.0"
Extension-Name: jakarta.servlet
Implementation-Vendor: Eclipse Foundation
Implementation-Vendor-Id: org.eclipse
Implementation-Version: 5.0.0-SNAPSHOT
Import-Package: jakarta.servlet,jakarta.servlet.annotation,jakarta.servl
 et.descriptor
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Specification-Vendor: Eclipse Foundation
Specification-Version: 5.0
Tool: Bnd-4.0.0.201805111645
```